lib/ocpp/v16/charge_point_configuration.cpp

```diff
@@ -2665,7 +2674,7 @@ std::optional<KeyValue> ChargePointConfiguration::getAllowChargingProfileWithout
         KeyValue kv;
         kv.key = "AllowChargingProfileWithoutStartSchedule";
         kv.readonly = false;
-        kv.value.emplace(std::to_string(allow.value()));
+        kv.value.emplace(std::to_string(static_cast<int>(allow.value())));
```

Comment:
Since this is a bool we should probably set it as a true or false string.
```suggestion
        kv.value = ocpp::conversions::bool_to_string(allow.value());
```